spellbook/models/nft/nft_schema.yml

```diff
@@ -1,34 +1,286 @@
+      - &amount_usd
+        name: amount_usd
         description:  "USD value of the trade at time of execution"
-      - name: unique_trade_id
-        description:  "Unique trade ID"
\ No newline at end of file
+      - &token_standard
+        name: token_standard
+        description:  "Token standard"
+      - &trade_type
+        name: trade_type
+        description: "Identify whether it was a single NFT trade or multiple NFTs traded"
+      - &number_of_items
+        name: number_of_items
+        description:  "Number of items traded"
+      - &trade_category
+        name: trade_category
+        description:  "How was this NFT traded ? (Direct buy, auction, etc...)"
+      - &evt_type
+        name: evt_type
+        description:  "Event type (Trade, Mint, Burn)"
+      - &seller
+        name: seller
+        description:  "Seller wallet address"
+      - &buyer
+        name: buyer
+        description:  "Buyer wallet address"
+      - &amount_original
+        name: amount_original
+        description:  "Traded amount in original currency"
+      - &amount_raw
+        name: amount_raw
+        description:   "Traded amount in original currency before decimals correction"
+      - &currency_symbol
+        name: currency_symbol
+        description:  "Symbol of original currency used for payment"
+      - &currency_contract
+        name: currency_contract
+        description:  "Contract address of original token used for payment, with ETH contract address swapped for WETH"
+      - &currency_contract_original
+        name: currency_contract_original
+        description:  "Contract address of original currency used for payment"
+      - &nft_contract_address
+        name: nft_contract_address
+        description:  "NFT contract address, only if 1 nft was transacted"
+      - &project_contract_address
+        name: project_contract_address
+        description:  "Contract address used by the project, in this case wyvern contract"
+      - &aggregator_name
+        name: aggregator_name
+        description:  "If the trade was performed via an aggregator, displays aggregator name"
+      - &aggregator_address
+        name: aggregator_address
+        description:  "If the trade was performed via an aggregator, displays aggregator address"
+      - &tx_hash
+        name: tx_hash
+        description:  "Transaction hash"
+      - &block_number
+        name: block_number
+        description: "Block number in which the transaction was executed "
+      - &tx_from
+        name: tx_from
+        description:  "Address that initiated the transaction"
+      - &tx_to
+        name: tx_to
+        description:  "Address that received the transaction"
+      - &platform_fee_amount_raw
+        name: platform_fee_amount_raw
+        description:  "Raw numerical amount for platform fees"
+        tests:
+          - dbt_utils.accepted_range:
+              min_value: 0
+              inclusive: true
+      - &platform_fee_amount
```

Comment:
Have some kind of `platform_fee_percentage` and `royalty_fee_percentage` columns been considered? I know they can be calculated through `platform_fee`/ `amount_original` and `royalty_fee`/`amount_original` but I could see those being useful additions for both runtime on large datasets and more importantly that extra bit of clarity for dune newbies

Reply:
I was hesitant about those ones, especially because I'm not a fan of having millions of rows filled with just '2' or '2.5' for platform fees. But I guess it doesn't hurt to add it, and this will be pretty relevant for most platforms across chains so I'll do that as well 🤝